### DISCUSSION TITLE
Change token recognition pattern to match Plex Scanner

### DISCUSSION THREAD

**Discussion by efalsken** [ZeroQI/Absolute-Series-Scanner#500] [Ideas]
The built-in plex scanner uses curly brackets for id matching tv series and movies.
```
TV/
    24 {tvdb-76290}
    3 Body Problem {tvdb-411959}
Movies/
    8MM (1999) {imdb-tt0134273}
    21 Jump Street (2012) {imdb-tt1232829}
```

It currently supports `tvdb`, `tmdb`, and `imdb` depending on library type.

I'd like to see this scanner updated to use `{anidb-####}` curly braces instead of the current `[anidb-####]`square brackets.
Maybe it can be a setting-switch for compatibility?

**Comment by ZeroQI** [comment]
https://support.plex.tv/articles/plexmatch/
Very fair feature request for consistency
Line 86 replace:
- \[ with [\[\{] at the beginning
- \] with [\]\}] at the end
Pull request would be great

**Reply by efalsken** [reply]
ohhhh, you're totally right. I was hella overthinking it. Will do some testing and PR soon.

**Comment by efalsken** [comment]
Converted to issue #501
